.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,77 @@
+name: ci-cd
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
for this as well- should this be main?

Reply:
this as well! thanks for pointing that out

Reply:
```suggestion
      - main
```